lib/vip/vip.go

```diff
@@ -472,7 +472,9 @@ func (client *Client) GetActiveTokens(userID string) ([]string, error) {
 	*/
 	var enabledTokenID []string
 	for _, credentialBinding := range response.Body.VipResponseGetUserInfo.CredentialBindingDetail {
-		fmt.Printf("\n%+v\n", credentialBinding)
+		if client.Debug {
+			log.Printf("\n%+v\n", credentialBinding)
```

Comment:
For the future, please add a logger to the library, so that the logger can be injected.